### PR TITLE
More docs cleanup for Orchestrator

### PR DESCRIPTION
**Pull request by scheyal**
Cleanup Orchestrator docs.

### CODE REVIEW

File: Orchestrator/README.md [hunk 1]
```diff
@@ -138,9 +138,20 @@ const myDialog = new AdaptiveDialog('myDialog').configure({
 
 ## Composer Integration
 
-\<TBD: This section is FYI in preparation for upcoming Composer functionality. It will be updated once ready.>
+Orchestrator has been can be used as recognizer in [Bot Framework Composer][19]. To specify Orchestrator as a dialog recognizer:
 
-Once the feature flag is enabled in Composer, it is possible to specify Orchestrator as a recognizer. For the most basic intent recognition cases, simply specify Orchestrator as the recognizer, and fill in the language data as you would for LUIS. For more advanced scenarios, such as dispatch orchestration, follow the steps above to import and tune up routing quality. 
+1. Enable Orchestrator feature in Composer's Application Settings
```
Comment (tsuwandy) [inline Orchestrator/README.md:143]: I assume user no longer needs to enable Orchestrator via feature flag?

File: Orchestrator/README.md [hunk 2]
```diff
@@ -138,9 +138,20 @@ const myDialog = new AdaptiveDialog('myDialog').configure({
...
+2. In desired dialog select Orchestrator the recognizer dropdown menu
```
Comment (tsuwandy) [inline Orchestrator/README.md:144]: select Orchestrator in the recognizer dropdown menu

File: Orchestrator/README.md [hunk 3]
```diff
@@ -138,9 +138,20 @@ const myDialog = new AdaptiveDialog('myDialog').configure({
 
 ## Composer Integration
 
-\<TBD: This section is FYI in preparation for upcoming Composer functionality. It will be updated once ready.>
+Orchestrator has been can be used as recognizer in [Bot Framework Composer][19]. To specify Orchestrator as a dialog recognizer:
```
Comment (CezaryMarcjan) [inline Orchestrator/README.md:141]: remove "has been"

**Review by CezaryMarcjan — APPROVED** [review]

**Review by tsuwandy — APPROVED** [review]

File: Orchestrator/README.md (hunk 1 above)
Comment (scheyal) [reply Orchestrator/README.md:143]: No. They do for this release.

File: Orchestrator/README.md (hunk 2 above)
Comment (scheyal) [reply Orchestrator/README.md:144]: I thought it'd be good to indicate that it is per dialog.

Comment (scheyal) [reply Orchestrator/README.md:144]: Ok. it works this way as well...

**Pull request merged**